tests/testthat/test-threads.R

```diff
@@ -8,159 +8,197 @@ if (not_on_cran()) {
   data_file_json <- test_path("resources", "data", "bernoulli.data.json")
 }
 
-test_that("using threads_per_chain without stan_threads set in compile() warns", {
+# test_that("using threads_per_chain without stan_threads set in compile() warns", {
```

Comment:
Just adding a reminder for us to turn these tests back on before we merge. I often forget that I've commented out tests.